src/charm.py

```diff
@@ -522,7 +520,6 @@ def _prometheus_config(self) -> str:
         if alerting_config:
             prometheus_config["alerting"] = alerting_config
 
-        # By default only monitor prometheus server itself
         default_config = {
```

Comment:
In order to make `_prometheus_config` simpler, shouldn't we extract this dictionary to a property?